qc.ipynb

```diff
@@ -588,6 +590,333 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "index = \"rx5day\"\n",
+    "aggr_var = \"max\"\n",
```

Comment:
I had to set `aggr_var` to `mean` instead of `max` to match the barplots that I generated, i.e. the figure I generated was named `barplot_mean_rx5day_Ketchikan.png` if that makes sense.

Reply:
Ahh yeah, didn't explain this well, but I wanted to leave this in here because we have shared some plots with john that were min/max instead of the mean, even though only the mean version was requested for the most recent round. I added a blurb about this into the QC notebook to explain this a but more.